zennit/rules.py

```diff
@@ -62,16 +62,31 @@ class AlphaBeta(LinearHook):
     beta: float, optional
         Multiplier for the negative output term.
     '''
-    def __init__(self, alpha=2., beta=1.):
+    def __init__(self, alpha=2., beta=1.): 
+        assert alpha > 0 and beta > 0
+        assert (alpha - beta) == 1
```

Comment:
I am not a fan of assertions outside of testing code.
If you must add this check, check it with an if-condition and raise a proper ValueError with a proper message

Reply:
Alright, changed!